### ISSUE TITLE
TypeError: lines.flatMap is not a function

### ISSUE THREAD

**Issue by pedrosimao** [alexeden/rpi-led-matrix#3]
While running the example `kitchen-sink.ts` I got this error:
```
at Function.linesToMappedGlyphs (/home/pi/msa-led-counter/src/layout-utils.ts:98:18)
    at /home/pi/msa-led-counter/examples/kitchen-sink.ts:61:23
```

While running `text-layout-cli.ts` I got this: 

```
/home/pi/msa-led-counter/examples/text-layout-cli.ts caught:  TypeError: lines.flatMap is not a function
    at Function.linesToMappedGlyphs (/home/pi/msa-led-counter/src/layout-utils.ts:98:18)
    at render (/home/pi/msa-led-counter/examples/text-layout-cli.ts:203:19)
    at /home/pi/msa-led-counter/examples/text-layout-cli.ts:210:5
    at process._tickCallback (internal/process/next_tick.js:68:7)
```

I am not sure why....

**Comment by alexeden** [comment]
Hey @pedrosimao, `.flatMap` is a relatively recent addition to JS and by extension, node. I'll either add a note to the documentation that the latest version of node should be used for the examples, or more likely just replace it with a `.map` and `.reduce`.

Thanks for letting me know!

**Comment by alexeden** [comment]
Hi again @pedrosimao, I've gone through and removed use of `.flatMap`--published as 1.5.0--so you should be good to go! Let me know if you have anymore issues.
